tests/e2e/utils/pages/setup-ads/setup-budget.js

```diff
@@ -12,6 +12,17 @@ export default class SetupBudget extends MockRequests {
 		this.page = page;
 	}
 
+	/**
+	 * Get budget recommendation tip section.
+	 *
+	 * @return {import('@playwright/test').Locator} The budget recommendation text row.
```

Comment:
This JSDoc description is not quite consistent.

Reply:
Looks like it has been updated. 👍